discretize/mixins/vtkInterface.py

```diff
@@ -58,12 +60,13 @@ def __treeMeshToVTK(mesh, models=None):
         # Adjust if result was 2D (voxels are pixels in 2D):
         VTK_CELL_TYPE = vtk.VTK_VOXEL
         if ptsMat.shape[1] == 2:
-            # TODO: assumes any 2D tree meshes are on the XY Plane
             # Add Z values of 0.0 if 2D
             ptsMat = np.c_[ptsMat, np.zeros(ptsMat.shape[0])]
             VTK_CELL_TYPE = vtk.VTK_PIXEL
         if ptsMat.shape[1] != 3:
             raise RuntimeError('Points of the mesh are improperly defined.')
+        # Rotate the points to the cartesian system
+        ptsMat = np.dot(ptsMat, mesh.rotMtx)
```

Comment:
Does this now rotate into 3D?

Reply:
I could be wrong about this... but this should rotate the points of a mesh from any orthogonal coordinate system to the Cartesian system given any `rotation_matrix`. 
In the prior lines, I ensure that the points are 3D before getting here. Thus you could have a 2D mesh where `axis_u` is defined as something other than `(1,0,0)` (similarly for `axis_v`) and the third dimension of points in that local frame will be set to zeros which will then get rotated to the Cartesian space.